app/src/main/java/com/xmartlabs/template/di/ViewModelKey.kt

```diff
@@ -0,0 +1,15 @@
+package com.xmartlabs.template.di
+
+import android.arch.lifecycle.ViewModel
+import dagger.MapKey
+import kotlin.reflect.KClass
+
+@MustBeDocumented
```

Comment:
why so? And shouldn't this class be public then?

Reply:
I think we can remove it, it's a copy paste from the google example